ra_aid/database/tests/ra_aid/database/test_connection.py

```
@@ -389,256 +401,253 @@ def test_init_db_sets_is_in_memory_attribute(cleanup_db):
+
 import pytest
-import peewee
 
-from ra_aid.database.connection import (
-    init_db, get_db, close_db, 
-    db_var, DatabaseManager, logger
-)
 
 
 @pytest.fixture
 def cleanup_db():
```

Ruff (0.8.2)

Redefinition of unused `cleanup_db` from line 22

(F811)